portfolio/src/main/webapp/index.html

```diff
@@ -111,4 +139,6 @@ <h3>Upload an image:</h3>
       /></a>
     </footer>
   </body>
+   <script src="https://www.gstatic.com/charts/loader.js"></script>
+   <script src="script.js"></script>
```

Comment:
These two <script> tags are both already present in the <head> block (where they belong, I think). Remove?